k8s/config/samples/k8s_v1alpha1_vineyardd.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: vineyard-system
 spec:
   image: docker.pkg.github.com/v6d-io/v6d/vineyardd:alpine-latest
-  version: 0.8.2
+  version: 0.9.0
```

Comment:
Use `latest` as the default.

Otherwise you will need to modify too much code when you bump up the release version....

Reply:
Make sense.